packages/api-v2/types.ts

```diff
@@ -2246,6 +2246,17 @@ type MT5AccountListResponse = {
 
 type TAccountList = NonNullable<AccountListResponse['account_list']>[number] & { excluded_until: Date };
 
+type TradingPlatformStatusRequest = {
+    trading_platform_status: 1;
+};
+
+type TradingPlatformStatusResponse = {
+    trading_platform_status: {
+        platform: 'mt5' | 'ctrader' | 'dxtrade';
```

Comment:
```suggestion
        platform: 'mt5' | 'ctrader' | 'dxtrade';
```
ideally you should put the type i proposed in the comment above `typeof CFD_PLATFORMS[keyof typeof CFD_PLATFORMS]` in a shared and reuse it here

Reply:
This works as well if you dont wanna use shared and re-use the types from reliable source

```suggestion
 platform: Exclude<
            NonNullable<
                TSocketEndpoints['trading_platform_accounts']['response']['trading_platform_accounts']
            >[number]['platform'],
            undefined
        >;
```